ios/.swiftformat

```diff
@@ -10,7 +10,5 @@
 
 --enable markTypes
 
---enable isEmpty
```

Comment:
Wydaje się rozsądna reguła nie mamy możliwości poprawienia?

https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#isEmpty

Czy może dać exclude na M3U8Kit jeśli błędy występują w tym kodzie? Albo suppress dokładnych miejsc gdzie to występuje a nie możemy/nie chcemy przerobić.